tests/scripts/protect-i18n.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want
+ * to block the upload of modified files in the "locale"
+ * folder to the repository through a git hook generated
+ * with the Husky library.
+ *
+ * Usage: $ node .git-hook/locale.js
+ *
+ * Happy releasing!
+ */
+
+'use strict'
+
```

Comment:
We still want Actions workflows to be able to change these files, so let's make the Actions environment exempt from this check by adding something like this toward the top of the file:

```js
if (process.env.CI) process.exit()
```

Reply:
In the Github Actions documentation (https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables) it refers to some predefined environment variables and there is the one you comment, process.env.CI, so just adding the line you propose would be enough.

Reply:
@zeke Done -> https://github.com/nodejs/nodejs.org/pull/3172/files#diff-6d229f474368307be4db86c19d5c3f7cR20